components/docs-chef-io/content/habitat/origin-rbac.md

```diff
@@ -0,0 +1,176 @@
++++
+title = "Origin Membership & RBAC"
+
+date = 2020-10-12T13:53:50-07:00
+draft = false
+
+[menu]
+  [menu.habitat]
+    title = "Origin Membership & RBAC"
+    identifier = "habitat/origins/origin-rbac Builder RBAC"
+    parent = "habitat/origins"
+    weight = 30
++++
+
+[\[edit on GitHub\]](https://github.com/habitat-sh/habitat/blob/master/components/docs-chef-io/content/habitat/builder-rbac.md)
+
+Prerequisites:
+
+- [Download the Chef Habitat CLI]({{< relref "install-habitat.md" >}})
+- [Create a Chef Habitat Builder account]({{< relref "builder-account" >}})
+- [Generate a personal access token]({{< relref "builder-profile#create-a-personal-access-token" >}})
+- [Create an origin]({{< relref "origins.md#create-origin" >}}) or accept an [invitation]({{< relref "#manage-origin-membership-with-hab-origin-invitations" >}}) to an existing origin
+- [Get origin keys]({{< relref "origins.md#origin-keys" >}})
+
+## Role-Based Access Control (RBAC) for Chef Habitat Builder (SaaS and on-prem)
```

Comment:
Should "on-prem" be capitalized? Should it be formalized to "On Premises" since "on-prem" is (to me, at least) a little slangy / jargony (or is that OK in this case)?

Reply:
It was a product decision of yore.

Reply:
![tenor-148759549](https://user-images.githubusercontent.com/207178/96140893-72fa7300-0ece-11eb-8b49-f5541c5b9d4f.gif)